lib/ddtrace/contrib/rails/core_extensions.rb

```diff
@@ -24,25 +24,19 @@ def patch_renderer
     end
 
     def patch_template_renderer(klass)
+      # rubocop:disable Metrics/BlockLength
       do_once(:patch_template_renderer) do
         klass.class_eval do
           def render_with_datadog(*args, &block)
```

Comment:
Can methods defined here be extracted to a module?

Reply:
In theory, they could. I didn't want to do that here because doing so would depend on `prepend`, we'd end up with version differences between 1.9.3 and 2.0.0+, and then everything would get more complicated, which would increase the risk of creating more bugs. We had just experienced exactly this when we refactored the ActionController patch into a module.

I would like to do this, but we should do it in a separate PR, because its more important we have a stable, verified fix to ship first. Then when that's out, we can clean up the implementation to make things look cleaner.

Reply:
Would moving only `def render_with_datadog` to a separate module still require 'prepend'?